Nowcasting concept tweaks (#310)
nfidd/nfidd: sessions/nowcasting.qmd

As we can see, even though it may much seem like the epidemic curve is going down, in fact in the final data set one can see that at the time symptom onsets were still increasing.
The apparent decline towards the present (indicated by a dashed vertical line) was caused by the delay in reporting.

::: {.callout-note}

> ## Why then, you might ask, not just plot the data by date of reporting which correctly showed the data to be still increasing and should, by definition, not be subject to future changes?

Reviewer: ```suggestion
## Plotting and analysing data by report date
Looking at the plots one might suggest plotting and analysing the data by date of reporting which correctly showed the data to be still increasing and which should, by definition, not be subject to future changes.
```
Just worried the long sentence will render weirdly.

This can sometimes be a sensible way to visualise the data.

> However, reporting might itself be subject to biases such as breaks during the weekend, holidays etc or reporting delays may vary over timel

Reviewer 2: ```suggestion
However, reporting might itself be subject to biases such as breaks during the weekend, holidays etc or reporting delays may vary over time
```

Reviewer: and a full stop

plot(proportion_reported)
```

> ::: {.callout-tip}

Reviewer: ```suggestion
::: {.callout-tip}
## The `plnorm` function
```
